test-case-reporting/test/e2e.test.ts

```diff
@@ -32,7 +31,9 @@ jest.mock('../src/db', () => ({
     }),
 }));
 
-describe.skip('end-to-end', () => {
+import {app} from '../app';
```

Comment:
You mock `dbConnect` after the module has already been imported. Since `require` caches results, that means any module that imports and uses `dbConnect` probably isn't getting the mock

Reply:
I don't believe that's the source of the issue -- when I log the database info, it says it is a sqlite3 database in memory.

Still, done moving the mock before the imports.

Reply:
Where do you log the DB info? In the test or the app code?

Reply:
It was a quick and dirty console.log in the app code; I removed it before pushing. Regardless, it's still not working despite the re-ordering; want to look at it together Monday?